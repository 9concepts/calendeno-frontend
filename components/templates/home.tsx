import { Layout } from "../layouts/layout";

export const HomeTemplate = (): JSX.Element => {
  return (
    <Layout>
      <div>
        <h1 role="heading" className="text-3xl font-bold underline">
          Hello
        </h1>
      </div>
    </Layout>
  );
};
