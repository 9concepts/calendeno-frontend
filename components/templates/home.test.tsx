import { render, screen } from "@testing-library/react";
import { Primary } from "./home.stories";

describe("HomeTemplate", () => {
  it("renders a homepage", () => {
    render(<Primary />);

    const heading = screen.getByRole("heading", {
      name: /Hello/i,
    });

    expect(heading).toBeInTheDocument();
  });
});

it("renders homepage unchanged", () => {
  const { container } = render(<Primary />);
  expect(container).toMatchSnapshot();
});
