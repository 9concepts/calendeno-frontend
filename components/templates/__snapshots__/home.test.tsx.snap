// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <div>
      <h1
        class="text-3xl font-bold underline"
        role="heading"
      >
        Hello
      </h1>
    </div>
  </div>
</div>
`;
